server/helpers/sms.js

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios';
+import { sanitizePhoneNumber, isValidNigerianPhoneNumber } from './funtions';
+
+const { KUDI_SMS_USERNAME, KUDI_SMS_PASSWORD, SMS_SENDERNAME } = process.env;
+
+const NODE_ENV = process.env.NODE_ENV || 'development';
+
+const KUDI_API_URL = 'https://account.kudisms.net/api/';
+
+export const sendSms = async ({ phoneNumbers, message }) => {
```

Comment:
Since we are not saving phoneNumbers as an array in the database and we are only dealing with a maximum of 2 phone numbers, I think it will be advisable to pass it as directly instead of converting it to an array.

Converting to an array will mean that we would have to convert phoneNumbers to an array everywhere we want to use them which is not a great idea.

Please note that this function will be updated in the future to accommodate sending SMS to foreign numbers and we need to make it easy to understand while maintaining best practices. 

```suggestion
export const sendSms = async ({ message, phoneNumber1, phoneNumber2 }) => {
```